docs/content/en/docs/reference/clusterspec/packages.md

```diff
@@ -0,0 +1,938 @@
+          Registry portion of an OCI address to the bundle<br/>
+        </td>
+        <td>true</td>
+      </tr><tr>
+        <td><b>repository</b></td>
+        <td>string</td>
+        <td>
+          Repository portion of an OCI address to the bundle<br/>
+        </td>
+        <td>true</td>
+      </tr></tbody>
+</table>
+
+
+#### PackageBundleController.status
+<sup><sup>[↩ Parent](#packagebundlecontroller)</sup></sup>
+
+
+
+PackageBundleControllerStatus defines the observed state of PackageBundleController.
+
+<table>
+    <thead>
+        <tr>
+            <th>Name</th>
+            <th>Type</th>
+            <th>Description</th>
+            <th>Required</th>
+        </tr>
+    </thead>
+    <tbody><tr>
+        <td><b>detail</b></td>
+        <td>string</td>
+        <td>
+          Detail of the state.<br/>
+        </td>
+        <td>false</td>
+      </tr><tr>
+        <td><b>state</b></td>
+        <td>enum</td>
+        <td>
+          State of the bundle controller.<br/>
+          <br/>
+            <i>Enum</i>: ignored, active, disconnected<br/>
+        </td>
+        <td>false</td>
+      </tr></tbody>
+</table>
+
+
+## PackageBundle Fields
+### PackageBundle
+PackageBundle is the Schema for the packagebundles API.
+
+<table>
+    <thead>
+        <tr>
+            <th>Name</th>
+            <th>Type</th>
+            <th>Description</th>
+            <th>Required</th>
+        </tr>
+    </thead>
+    <tbody><tr>
+      <td><b>apiVersion</b></td>
+      <td>string</td>
+      <td>packages.eks.amazonaws.com/v1alpha1</td>
+      <td>true</td>
+      </tr>
+      <tr>
+      <td><b>kind</b></td>
+      <td>string</td>
+      <td>PackageBundle</td>
+      <td>true</td>
+      </tr>
+      <tr>
+      <td><b><a href="https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#objectmeta-v1-meta">metadata</a></b></td>
+      <td>object</td>
+      <td>Refer to the Kubernetes API documentation for the fields of the `metadata` field.</td>
+      <td>true</td>
+      </tr><tr>
+        <td><b><a href="#packagebundlespec">spec</a></b></td>
+        <td>object</td>
+        <td>
+          PackageBundleSpec defines the desired state of PackageBundle.<br/>
+        </td>
+        <td>false</td>
+      </tr><tr>
+        <td><b><a href="#packagebundlestatus">status</a></b></td>
+        <td>object</td>
+        <td>
+          PackageBundleStatus defines the observed state of PackageBundle.<br/>
+        </td>
+        <td>false</td>
+      </tr></tbody>
+</table>
+
+
+#### PackageBundle.spec
+<sup><sup>[↩ Parent](#packagebundle)</sup></sup>
+
+
+
+PackageBundleSpec defines the desired state of PackageBundle.
+
+<table>
+    <thead>
+        <tr>
+            <th>Name</th>
+            <th>Type</th>
+            <th>Description</th>
+            <th>Required</th>
+        </tr>
+    </thead>
+    <tbody><tr>
+        <td><b><a href="#packagebundlespecpackagesindex">packages</a></b></td>
+        <td>[]object</td>
```

Comment:
Are the `[]` characters intentional? I assume they are since I see them in other places.

Reply:
They are intentional. There is a different `object` type without the `[]`, which maps to a use case similar to `PackageBundle.status.spec`. For the one with `[]object`, it maps to a use case similar to `PackageBundle.spec.packages[index]`.